src/test/end-to-end/shared/testingSupportApi.ts

```diff
@@ -1,7 +1,7 @@
 import superagent from 'superagent';
```

Comment:
This file will get bigger in the future. I wonder if it is better to split out the support methods into separate files, one for data-management API calls, one for subscription-management API call, etc., so there are clear distinction of functionality within each file. It is up to you though so just a thought.

Reply:
I don't think we are going to add too many methods to this. There will be no need for separate files.